apps/condo/domains/organization/schema/OrganizationEmployeeRole.js

```diff
@@ -79,6 +79,10 @@ const OrganizationEmployeeRole = new GQLListSchema('OrganizationEmployeeRole', {
             type: Checkbox,
             defaultValue: false,
         },
+        canManageHelps: {
+            type: Checkbox,
+            defaultValue: false,
```

Comment:
In initial permissions for roles it states 'true', maybe here also should be 'true'?

Reply:
It's OK for new Features